domains/abhay.json

```
@@ -0,0 +1,12 @@
+{
+  "description": "Abhay's portfolio website",
+  "repo": "https://github.com/daggron/daggron.github.io",
+  "owner": {
+    "username": "Daggron",
+    "email": "returnofking04@gmail.com"
+  },
+  "record": {
+    "CNAME": "codeinator.me",
+    "URL": "http://codeinator.me"
```

You can't have a cname and a url redirection to the same domain.

If you just want to redirect to codeinator.me
```suggestion
    "URL": "http://codeinator.me"
```

If you just want a CNAME record,
```suggestion
    "CNAME": "codeinator.me"
```

You can refer to the [docs](https://github.com/is-a-dev/register/blob/main/API.md) for more info. If you have any questions, feel free to post it as an issue.

---

I have updated this, to abhay.is-a.dev. Please have a look